pkgs/development/libraries/gobject-introspection/wrapper.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, stdenv
+, makeWrapper
+, buildEnv
+, writeText
+, runCommand
+, buildPackages
+, gobject-introspection-unwrapped
+, pkgsHostHost
+, pkgsBuildBuild
+, targetPackages
+}:
+
+# to build, run
+# `nix build ".#pkgsCross.aarch64-multiplatform.buildPackages.gobject-introspection"`
+gobject-introspection-unwrapped.overrideAttrs (_previousAttrs: {
```

Comment:
We're overriding? Can we make a fresh derivation like the other wrappers?

Reply:
I tried that but dirs would have to be copied to the wrapper and paths rewritten and then split again, so this is the simplest and most maintainable way.